src/utils/getExplorerLink.ts

```diff
@@ -27,14 +27,14 @@ export function getExplorerLink(chainId: number, data: string, type: ExplorerDat
   if (chainId === SupportedChainId.ARBITRUM_ONE) {
     switch (type) {
       case ExplorerDataType.TRANSACTION:
-        return `https://explorer.arbitrum.io/tx/${data}`
+        return `https://arbiscan.io/tx/${data}`
       case ExplorerDataType.ADDRESS:
       case ExplorerDataType.TOKEN:
-        return `https://explorer.arbitrum.io/address/${data}`
+        return `https://arbiscan.io/address/${data}`
```

Comment:
should we use the /token/ path instead? arbiscan provides some more helpful information in that case
https://arbiscan.io/token/0x354a6da3fcde098f8389cad84b0182725c6c91de